src/app/academic-year/academic-year.component.scss

```diff
@@ -10,3 +9,7 @@
 .switch-years {
   width: 350px;
 }
+
+#deviation-sliders p {
+  margin-top: 20px;
+}
```

Comment:
bootstrap provides classes like "mt-*"
where
m -> margin
t -> top
* -> the number of px

you could avoid use CSS and just use `<p class="mt-20">` (same for padding using pt-*)